R/pkg/R/functions.R

```diff
@@ -2226,8 +2226,8 @@ setMethod("from_json", signature(x = "Column", schema = "characterOrstructType")
           })
 
 #' @details
-#' \code{from_utc_timestamp}: Given a timestamp, which corresponds to a certain time of day in UTC,
-#' returns another timestamp that corresponds to the same time of day in the given timezone.
+#' \code{from_utc_timestamp}: Given a timestamp like '2017-07-14 02:40:00.0', interprets it as a time in UTC, and
```

Comment:
I think lintr does not recognise this case but I believe these lines should fit in 100 character lengths ..